Source/Models/SwishError.swift

```diff
@@ -0,0 +1,26 @@
+import Foundation
+import Argo
+
+public enum SwishError {
+  case SessionError(NSError)
+  case ServerError(code: Int, json: AnyObject)
+  case JSONSerializationError(NSError)
+  case ArgoError(DecodeError)
+}
+
+extension SwishError: ErrorType { }
+
+public extension SwishError {
+  var rawError: NSError {
```

Comment:
Primarily, this is used to transform `SwishError` -> `NSError` for the default `transformError` implementation.